tests/TwilioFakeTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace Aloha\Twilio\Tests;
```

Comment:
this keeps this class out of production

Reply:
Agree. I chose that namespace because the other two tests/ files were using it. For OSS, I tend to adhere to the original author's opinions instead of rocking the boat.